package.json

```diff
@@ -27,6 +27,7 @@
     "lint-md": "remark -q *.md src/**/*.md configs/*.md docs/**/*.md",
     "lint-fix": "yarn lint-js -- --fix",
     "test": "jest --forceExit",
+    "test-coverage": "yarn test -- --coverage",
```

Comment:
sure - this is probably fine. we can always update this later when we're using flow as well